src/app/static/src/js/components/runWorkflow/runWorkflowRun.vue

```diff
@@ -1,73 +1,192 @@
-            <div class="col-sm-4">
-                <select class="form-control" id="run-workflow-changelog" >
-                    <option value="interim update">
-                        Interim update
-                    </option>
-                </select>
-            </div>
+        <div v-if="showInstances">
+            <instances :instances="runMetadata.metadata.instances"
+                       :custom-style="changelogStyle"
+                       @selectedValues="handleInstancesValue"/>
         </div>
-        <div id="workflow-changelog-type-div" class="form-group row">
-            <label class="col-sm-4 col-form-label text-left">Changelog type</label>
-            <div class="col-sm-4">
-                <input type="text" class="form-control" id="run-workflow-changelog-type" placeholder="Internal">
-            </div>
+        <div v-if="runMetadata">
+            <change-log :show-changelog="handleShowChangeLog"
+                        :changelog-type-options="this.runMetadata.metadata.changelog_types"
+                        :custom-style="changelogStyle"
+                        @changelogMessage="handleChangeLogMessageValue"
+                        @changelogType="handleChangeLogTypeValue">
+            </change-log>
         </div>
-        <div id="workflow-tags-div" class="form-group row">
+        <div v-if="switchV2FeatureComponentOn" id="workflow-tags-div" class="form-group row">
             <label class="col-sm-4 col-form-label text-left">Report version tags</label>
             <div class="col-sm-4">
-                <input type="text" @input="validateStep" class="form-control" id="run-workflow-report-version-tags" placeholder="interim estimates">
+                <input type="text" class="form-control" id="run-workflow-report-version-tags">
             </div>
         </div>
-        <div id="workflow-completion-div" class="form-group row">
+        <div v-if="switchV2FeatureComponentOn" id="workflow-completion-div" class="form-group row">
             <label class="col-sm-4 col-form-label text-left">Only commit reports on workflow completion</label>
             <div id="run-workflow-ticked" class="col-sm-4">
                 <p>ticked</p>
             </div>
         </div>
+        <error-info :default-message="defaultMessage" :api-error="error"></error-info>
     </div>
 </template>
 
 <script lang="ts">
-import Vue from "vue"
-import {RunWorkflowMetadata} from "../../utils/types";
+    import Vue from "vue"
+    import {ChangelogStyle, RunReportMetadata, RunWorkflowMetadata, WorkflowSummary} from "../../utils/types";
+    import {api} from "../../utils/api";
+    import ErrorInfo from "../../../js/components/errorInfo.vue";
+    import ChangeLog from "../../../js/components/runReport/changeLog.vue";
+    import Instances from "../../../js/components/runReport/instances.vue";
+
+    interface Props {
+        workflowMetadata: RunWorkflowMetadata | null
+        isRerun: boolean
```

Comment:
This is marginal, but might more sense at this level for the prop to be called `disableRename` or something. This component doesn't really need to understand the concept of re-running.